packages/terra-table/src/TableCell.jsx

```diff
@@ -12,11 +12,14 @@ const propTypes = {
 const TableCell = ({
   content,
   ...customProps
-}) => (
-  <td {...customProps} data-class="cell">
-    {content}
-  </td>
-);
+}) => {
+  const dataClassNames = customProps['data-class'] ? `cell ${customProps['data-class']}` : 'cell';
```

Comment:
I agree with @StephenEsser, if we want to keep the class names to represent the structure, we should recommend using name-spaced data attributes.